packages/aws-cdk-lib/aws-apigateway/lib/method.ts

```diff
@@ -199,6 +199,11 @@ export class Method extends Resource {
         `which is different from what is required by the authorizer [${authorizer.authorizationType}]`);
     }
 
+    // When AuthorizationType is None, there shouldn't be any AuthorizationScope since AuthorizationScope should only
+    // be applied to COGNITO_USER_POOLS AuthorizationType.
```

Comment:
If I understand this correctly, `authorizationScopes` should not be exposed to users? Should we deprecate it and warn against its usage?

Reply:
I may miss something, why should we not expose it?

Reply:
I misunderstood the comment, apologies. Shouldn't we warn or throw instead of silently overriding the user's selection?

Reply:
Good call, will add a warning.